TSIS2.WebResources/TSIS2.WebResources.csproj

```diff
@@ -150,7 +150,6 @@
     <Content Include="Webresources\js\surveycreator.js" />
     <Content Include="Webresources\js\surveyRender.js" />
     <Content Include="Webresources\js\tsis2findingwidget.js" />
-    <Content Include="Webresources\js\TSIS2OversightActivity.js" />
```

Comment:
Probably won't be removed after deploy so please double check and remove it manually.